db/schema.rb

```diff
@@ -170,8 +171,8 @@
     t.integer "accrediting_provider_id"
     t.boolean "exposed_in_find"
     t.boolean "open_on_apply", default: false, null: false
-    t.string "study_mode", limit: 1, default: "F", null: false
     t.integer "recruitment_cycle_year", null: false
+    t.string "study_mode", limit: 1, default: "F", null: false
```

Comment:
How do we resolve this? The db migration for study_mode has an earlier timestamp than the one for recruitment_cycle_year, but the recruitment_cycle_year code was merged first. Any new contributor will have study_mode first, recruitment_cycle_year second, do you think you could rollback the migrations and re-apply?

Reply:
Good spot, yes!